Mathlib/Algebra/Group/Pi/Basic.lean

```diff
@@ -257,36 +257,21 @@ theorem extend_mul [Mul γ] (f : α → β) (g₁ g₂ : α → γ) (e₁ e₂ :
     Function.extend f (g₁ * g₂) (e₁ * e₂) = Function.extend f g₁ e₁ * Function.extend f g₂ e₂ := by
   classical
   funext x
-  simp only [extend_def, Pi.mul_apply, apply_dite₂, dite_eq_ite, ite_self]
--- Porting note: The Lean3 statement was
--- `funext <| λ _, by convert (apply_dite2 (*) _ _ _ _ _).symm`
--- which converts to
--- `funext fun _ => by convert (apply_dite₂ (· * ·) _ _ _ _ _).symm`
--- However this does not work, and we're not sure why.
+  simp [Function.extend_def, apply_dite₂]
```

Comment:
Flagging this for reviewers (e.g. @jcommelin) to double-check that these notes are not actionable/not worth keeping.

Reply:
The new version seems just better, so let's remove the porting note.